pom.xml

```diff
@@ -86,6 +86,8 @@
 
         <!-- TODO Are these correct versions? -->
         <bouncycastle.pkixfips.version>1.0.7</bouncycastle.pkixfips.version>
+        <!-- 1.0.2.4 exhibits class loading issues -->
+        <bouncycastle.bcfips.version>1.0.2.3</bouncycastle.bcfips.version>
```

Comment:
Question: did you tried the other way around?

I mean instead of overriding `bouncycastle.bcfips.version` with the lower version than quarkus bom, the alternative might be to update`bouncycastle.pkixfips.version` to newer version?

If it doesn't work, is it possible to create follow-up issue for look into this? It can be nice to use the latest BCFIPS libraries to avoid risk of having some security issues...

Reply:
@mposolda there are no newer versions of pkixfips.

> If it doesn't work, is it possible to create follow-up issue for look into this?

See #26468 

> It can be nice to use the latest BCFIPS libraries to avoid risk of having some security issues...

Yes, it looks like a vulnerability was reported against 1.0.2.3